src/Symfony/Component/VarDumper/Dumper/CliDumper.php

```diff
@@ -58,8 +58,8 @@ public function __construct($output = null, $charset = null, $flags = 0)
     {
         parent::__construct($output, $charset, $flags);
 
-        if ('\\' === DIRECTORY_SEPARATOR && false !== @getenv('ANSICON')) {
-            // Use only the base 16 xterm colors when using ANSICON
+        if ('\\' === DIRECTORY_SEPARATOR && 'ON' !== @getenv('ConEmuANSI') && 'xterm' !== @getenv('TERM')) {
+            // Use only the base 16 xterm colors when using ANSICON or standard Windows 10 CLI
```

Comment:
Because standard Windows 10 does not support 256 colors, but only 16 like ANSICON.
But if we use ConEmuANSI or xterm we have 256 colors.